src/libs/reportUtils.js

```diff
@@ -162,12 +162,13 @@ function findLastAccessedReport(reports, ignoreDefaultRooms) {
 /**
  * Whether the provided report is an archived room
  * @param {Object} report
+ * @param {String} report.chatType
  * @param {Number} report.stateNum
  * @param {Number} report.statusNum
  * @returns {Boolean}
  */
 function isArchivedRoom(report) {
-    if (!isDefaultRoom(report)) {
+    if (!isChatRoom(report) && !isPolicyExpenseChat(report)) {
         return false;
     }
 
```

Comment:
Can't comment on the line below this one, but we should change the `2` in the comparison to some `CONST` like

```js
return report.statusNum === CONST.REPORT.STATUS_CLOSED && report.stateNum === CONST.REPORT.STATE_MANUALREIMBURSED;
```